package.json

```diff
@@ -1,13 +1,17 @@
 {
   "private": true,
   "scripts": {
-    "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "eslint src && pulp build -- --censor-lib --strict"
+    "clean": "rimraf output && rimraf .pulp-cache && rimraf .spago",
+    "build": "spago build",
+    "test": "spago build && purescm run --main Test.Main"
   },
   "devDependencies": {
```

Comment:
I don't think we need these either. CI installs all tools by hand and this list includes many JS tools that are not needed.